.github/workflows/build-plugin-archive.yml

```
@@ -147,6 +149,11 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: ${{ needs.checkout-dependencies.outputs.artifact }}
+      - name: Setup PHP with tools
```

In all other workflows, we name this step "Set up PHP", regardless of whether or not we use it to install additional tools. Also, there's a new line above each step.
```suggestion

      - name: Set up PHP
```